mongo_connector/oplog_manager.py

```diff
@@ -74,11 +75,14 @@ def __init__(self, primary_client, doc_managers,
         # The set of namespaces to process from the mongo cluster.
         self.namespace_set = kwargs.get('ns_set', [])
 
+        # The set of namespaces not to process from the mongo cluster.
+        self.ex_namespace_set = kwargs.get('ex_ns_set', [])
+
         # The set of gridfs namespaces to process from the mongo cluster
         self.gridfs_set = kwargs.get('gridfs_set', [])
 
-        # The dict of source namespaces to destination namespaces
-        self.dest_mapping = kwargs.get('dest_mapping', {})
+        # The structure handling dest_mappings dynamically
+        self.dest_mapping = dest_mapping_stru
```

Comment:
I don't think we need to change the constructor signature. We can still pull "dest_mapping" out of kwargs. I don't think we need to change the parameter name either.